lib/prawn/text/formatted/arranger.rb

```diff
@@ -212,6 +219,8 @@ def font_style(styles)
 
         private
 
+        attr_accessor :finalized
```

Comment:
This seem to be deprecated in Ruby 2. Doesn't it warn you?

`protected` seem to be fine for `attr_accessor`, though.